integration-test/fixture.ts

```diff
@@ -1,5 +1,34 @@
 import { test as base, expect } from "@playwright/test";
 
+declare global {
+  namespace PlaywrightTest {
+    interface Matchers<R, T> {
+      toBeInitiallyLoading(expectedState: boolean): R;
+    }
+  }
+}
+
+expect.extend({
+  async toBeInitiallyLoading(
```

Comment:
This tests for the loading indicator **immediately** instead of waiting up to 5 seconds for it to appear or disappear, which will accidentally succeed in a lot of cases.